src/cube.ml

```diff
@@ -0,0 +1,112 @@
+open! Import
+
+type t = floatarray
+(* lower left x, upper right x, lower left y, upper right y, lower left z, upper right z*)
```

Comment:
I think it is a bit ambiguous for the code reader to understand the points. It would be better to write "min", "max" value of a coordinate.